scan.py

```diff
@@ -197,11 +203,48 @@ def sns_scan_results(
         },
     )
 
+def call_api_with_scan_results(s3_object, result):
+    if AV_STATUS_CALL_API_MAP is None:
+        return
+
+    try:
+        av_status_call_api_map = str(AV_STATUS_CALL_API_MAP)
+        av_status_call_api_map = eval(AV_STATUS_CALL_API_MAP.replace('"', '\"'))
+
+        av_status_call_api = av_status_call_api_map[s3_object.bucket_name]
+
+        av_status_call_api_url = av_status_call_api['url']
+        av_status_call_api_username = av_status_call_api['username']
+        av_status_call_api_password = av_status_call_api['password']
+    except ValueError:
+        print("Skipping API call with scan results, encountered error while parsing AV_STATUS_CALL_API_MAP")
```

Comment:
An email notification here as well would be good !

Reply:
Agreed. 
Although, I suggest having this integration as a separate task, since it would impact the timeline for the engineering security ask. 

cc: @Harishgolla